src/boot/stylesheets/main.scss

```diff
@@ -44,6 +44,18 @@ body {
     	color: $gray;
     }
 
+    button {
+		background-color: transparent;
```

Comment:
I guess how do we feel about links that perform actions?

While it's not ideal, it also feels strange that we're setting some base styling for buttons to look like links.